cmd/minikube/cmd/delete.go

```diff
@@ -145,6 +146,37 @@ func deleteContainersAndVolumes(ctx context.Context, ociBin string) {
 	}
 }
 
+// kicbaseImage fetches images based on container label
+func kicbaseImages(ctx context.Context, ociBin string) ([]string, error) {
+	if _, err := exec.LookPath(ociBin); err != nil {
+		return nil, nil
+	}
+	delLabel := fmt.Sprintf("%s=%s", oci.CreatedByLabelKey, "true")
+	return oci.ListImagesByContainerLabel(ctx, ociBin, delLabel)
+}
+
+// printDeleteImagesCommand prints command which remove images
+func printDeleteImagesCommand(ociBin string, imageNames []string) {
+	if _, err := exec.LookPath(ociBin); err != nil {
+		return
+	}
+
+	if len(imageNames) > 0 {
+		out.Styled(style.Command, `{{.ociBin}} rmi {{.images}}`, out.V{"ociBin": ociBin, "images": strings.Join(imageNames, "")})
```

Comment:
Missed this, but I'm assuming we should have a space between the images?

Reply:
That's correct.